cmdstanpy/stanfit/mcmc.py

```
@@ -97,8 +97,9 @@ def __init__(
         self._max_treedepths: np.ndarray = np.zeros(
             self.runset.chains, dtype=int
         )
+        self._chain_timing: List[Optional[Dict[str, float]]] = []
```

The `Optional` here means some elements of this list can be `None`, but if I'm reading the parsing code correctly that case is impossible. Either we should let the parser fail without raising an error or change the type here and on the method

---

good call - removed Optional